shot-consumer-flavors/scripts/create_emulator.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/bash
 
-if $ANDROID_HOME/tools/android list avd | grep -q Nexus_5X_API_26; then
```

Comment:
What's the reason behind the changes in the emulator and the SDCard image names?

Reply:
Just to be able to create different emulators with different sdcards for different consumers.